atom/github
# Update to @atom/mocha-test-runner and mocha@6.0.2

**Please be sure to read the [contributor's guide to the GitHub package](https://github.com/atom/github/blob/master/CONTRIBUTING.md) before submitting any pull requests.**

### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* Suggestion: You can use checklists to keep track of progress for the sections on metrics, tests, documentation, and user research.

### Description of the Change

Change our test runner dependency from [@smashwilson/atom-mocha-test-runner]() to [@atom/mocha-test-runner](). Update Mocha to match the version from the new runner.

### Screenshot/Gif

<img width="1600" alt="hey look it still works" src="https://user-images.githubusercontent.com/17565/53508871-9e75e300-3a88-11e9-9514-408ac6f744c8.png">

### Alternate Designs

_N/A_

### Benefits

Most immediately, it'll address some lagging dependencies and transitive dependencies, and fix #1965.

### Possible Drawbacks

There's the possibility of breaking changes. This is a major-version bump for Mocha.

### Applicable Issues

Fixes #1965.

### Metrics

_N/A_

### Tests

* Verify that the graphical test runner works.
* Verify that the headless test runner works.
* Verify that our istanbul integration still works (by ensuring that the CodeCov results look correct).

### Documentation

_N/A_

### Release Notes

_N/A_

### User Experience Research (Optional)

_N/A_
